contracts/axelar-gateway/src/testutils.rs

```diff
@@ -53,6 +53,17 @@ pub fn setup_gateway<'a>(
     (signer_set, client)
 }
 
+pub fn setup_env<'a>(
+    previous_signers_retention: u32,
+    num_signers: u32,
+) -> (Env, TestSignerSet, AxelarGatewayClient<'a>) {
+    let env = Env::default();
+    env.mock_all_auths();
```

Comment:
Mocking auth here might affect tests that check the auth. Double check the use

Reply:
there are tests that depend on auth being mocked (although it's not necessary for all of them), so I think it's fine to keep

Reply:
let's mock it explicitly for the tests that need it. you can use `assert_invoke_auth_ok` to avoid having to mock auth completely instead for some of the tests that need auth as well. This can make the test more precise and avoids slipping in an auth bug due to mocking